scripts/package/service/systemd/observiq-otel-collector.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
```

Comment:
Do you mind moving this into the top-level `service` directory? We added the macOS plist file and I think they should be next to eachother.